nss-day-cohort-51/nss-day-cohort-51.github.io
# Ss feature styling

# Description - updated modal

## Type of change - updated modal


# Testing Instructions
Fetch --all then checkout into ss-feature-styling and make sure modal is updated

# Checklist:
